Source/WTF/wtf/Compiler.h

```diff
@@ -281,6 +281,18 @@
 #define NOT_TAIL_CALLED
 #endif
 
+/* MUST_TAIL_CALL */
+
+#if !defined(MUST_TAIL_CALL) && defined(__cplusplus) && defined(__has_cpp_attribute)
+#if __has_cpp_attribute(clang::musttail)
+#define MUST_TAIL_CALL [[clang::musttail]]
```

Comment:
this does not work well with clang 17 on arm backend. see https://github.com/llvm/llvm-project/issues/67767#issuecomment-1741736944